### PR TITLE
TE-1462 Adding test_read_only_admin_can_read_the_dataset_ui and fixed storage and Dataset issues.

### PR DESCRIPTION
**Pull request by ericbsd**
image

image

image

### CODE REVIEW

File: keywords/ssh/common.py [hunk 1]
```diff
@@ -186,9 +187,12 @@ def set_host_ssh_key_and_enable_ssh_on_the_nas(cls, username: str):
         :param username: the username to set the SSH public key on the NAS
         """
         cls.create_ssh_key()
+        assert path.exists(path.expanduser(f'{shared_config["KEYPATH"]}.pub'))
+        print(cls.get_ssh_pub_key())
```
Comment (jordan-mcgillivray005) [inline keywords/ssh/common.py:191]: Should probably add some text to the print to explain this in console.

File: keywords/ssh/common.py [hunk 2]
```diff
@@ -186,9 +186,11 @@ def set_host_ssh_key_and_enable_ssh_on_the_nas(cls, username: str):
         :param username: the username to set the SSH public key on the NAS
         """
         cls.create_ssh_key()
+        assert path.exists(path.expanduser(f'{shared_config["KEYPATH"]}.pub')) is True
         assert API_PUT.set_user_ssh_public_key(username, cls.get_ssh_pub_key()).status_code == 200
         assert API_POST.start_service('ssh').status_code == 200
         assert API_PUT.enable_service_at_boot('ssh').status_code == 200
+        assert path.exists(path.expanduser(f'{shared_config["KEYPATH"]}.pub'))
```
Comment (jordan-mcgillivray005) [inline keywords/ssh/common.py:193]: This line also needs an is True at the end

File: test_cases/webui/datasets/test_access_dataset_space_management.py [hunk 3]
```diff
@@ -71,9 +82,11 @@ def test_access_dataset_managing_group_quotas(self, data):
         Datasets.select_dataset(data["dataset"])
         # Verify the group quotas page for the given dataset opens.
         assert Datasets.is_space_management_card_visible()
```
Comment (jordan-mcgillivray005) [inline test_cases/webui/datasets/test_access_dataset_space_management.py:84]: This two lines also needs an is True at the end

File: keywords/ssh/common.py (hunk 2 above)
Comment (ericbsd) [reply keywords/ssh/common.py:193]: removed the duplicated

**Review by markbainix — APPROVED** [review]

**Pull request merged**